command.go

```diff
@@ -43,6 +45,25 @@ func (c *Cmd) DeleteCmd(name string) {
 	delete(c.children, name)
 }
 
+// CommandForName returns the subcommand with matching name or alias.
+func (c *Cmd) CommandForName(name string) *Cmd {
```

Comment:
Let's make this a private method and rename it to something a bit better. e.g. `findChild`